src/main/java/com/cleanroommc/groovyscript/sandbox/security/SandboxSecurityManager.java

```diff
@@ -0,0 +1,92 @@
+package com.cleanroommc.groovyscript.sandbox.security;
+
+import com.cleanroommc.groovyscript.GroovyScript;
+import sun.misc.Unsafe;
+
+import java.io.FilePermission;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.security.Permission;
+
+public class SandboxSecurityManager extends SecurityManager {
```

Comment:
`FMLSecurityManager.java:54` throws `new SecurityException("Cannot replace the FML security manager")` whenever scripts are loaded or reloaded